doc/reference.md

```diff
@@ -275,8 +291,24 @@ constexpr optional<size_t> enum_index() noexcept;
 
   ```cpp
   constexpr auto color_index = magic_enum::enum_index(Color::BLUE);
-  // color_index -> color_index.value() -> 1
-  // color_index -> color_index.has_value() -> true
+  // color_index.value() -> 1
+  // color_index.has_value() -> true
+  ```
+
+```cpp
+template <auto V>
+constexpr size_t enum_index() noexcept;
```

Comment:
Just a stylistic inconsistency:
At `enum_name` the documentation of the function variants was merged. These variants (`enum_index` and `enum_value`) can be merged too.